### DISCUSSION TITLE
Issue with upgrade from 1.10.3 to 2.6.3

### DISCUSSION THREAD

**Discussion by VijayBShinde** [apache/airflow#32602] [General]
Hello Community,

0

let me explain Airflow upgradation and migration story here. I am using airflow version 1.10.3 with python 2.7 version to run my shell scripts, Apache Spark jobs etc. All jobs are working fine in current airflow version.

I recently started upgrading the airflow to latest stable airflow version 2.6.3 on Python 3.8 version and set up 2 node airflow cluster on two different Linux server.

Also, have set up all the dependencies python packages, jars, configuration files etc.

Now, When I am running jobs from new airflow, the job is getting terminated with AirflowException. Here is the details exception.

> 
> [2023-07-14, 09:21:42 UTC] {local_task_job_runner.py:271} WARNING - Recorded pid 24036 does not match the current pid 24058
> [2023-07-14, 09:21:42 UTC] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 24058. PIDs of all processes in the group: [24324, 24327, 24340, 24341, 24342, 24351, 24352, 24353, 24354, 24058]
> [2023-07-14, 09:21:42 UTC] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 24058
> [2023-07-14, 09:21:42 UTC] {taskinstance.py:1517} ERROR - Received SIGTERM. Terminating subprocesses.
> [2023-07-14, 09:21:42 UTC] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
> [2023-07-14, 09:21:42 UTC] {taskinstance.py:1824} ERROR - Task failed with exception
> Traceback (most recent call last):
>   File "/usr/lib/python3.8/site-packages/airflow/operators/bash.py", line 201, in execute
>     result = self.subprocess_hook.run_command(
>   File "/usr/lib/python3.8/site-packages/airflow/hooks/subprocess.py", line 91, in run_command
>     for raw_line in iter(self.sub_process.stdout.readline, b""):
>   File "/usr/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1519, in signal_handler
>     raise AirflowException("Task received SIGTERM signal")
> airflow.exceptions.AirflowException: Task received SIGTERM signal
> [2023-07-14, 09:21:42 UTC] {taskinstance.py:1345} INFO - Marking task as FAILED
> 

To solve above issue tried below option.

Disabled mini schedular : schedule_after_task_execution = False

dagrun_timeout : this set to 1360 min

Any suggestions or assistance provided in addressing this issue would be greatly appreciated.

**Comment by raphaelauv** [comment]
https://github.com/apache/airflow/discussions/24031#discussioncomment-5731704
do 

1.10.3 -> 1.10.15 ->  2.2.5 -> 2.5.3 -> 2.6.3

**Reply by VijayBShinde** [reply]
Thanks @raphaelauv for the reply. Could you please share details what are you trying explain

**Reply by pavelpi** [reply]
@raphaelauv we are experiencing the same problem with 2.6.2. We are installing a new system so the recommended upgrade path does not make sense.
